src/game/behaviors/scuttlebug.inc.c

```diff
@@ -27,14 +27,18 @@ void bhv_scuttlebug_loop(void) {
     UNUSED s32 unused;
     f32 sp18;
     obj_update_floor_and_walls();
-    if (o->oSubAction != 0
-        && obj_set_hitbox_and_die_if_attacked(&sScuttlebugHitbox, SOUND_OBJ_DYING_ENEMY1,
-                                              o->oScuttlebugUnkF4))
-        o->oSubAction = 3;
-    if (o->oSubAction != 1)
+    if (o->oSubAction != 0 && o->oSubAction != 1
+        && obj_set_hitbox_and_die_if_attacked_bingo(&sScuttlebugHitbox, SOUND_OBJ_DYING_ENEMY1,
+                                              o->oScuttlebugUnkF4, BINGO_UPDATE_KILLED_SCUTTLEBUG))
+        o->oSubAction = 4;
+    if (o->oSubAction != 2)
         o->oScuttlebugUnkF8 = 0;
     switch (o->oSubAction) {
         case 0:
+            o->oBingoId = get_unique_id(BINGO_UPDATE_KILLED_SCUTTLEBUG, o->oPosX, o->oPosY, o->oPosZ);
+            o->oSubAction = 1;
+            break;
```

Comment:
I think this technically changes the behavior of Scuttlebugs by adding an extra frame before they are initialized. I generally try to keep game behavior as identical as possible to vanilla SM64. Instead, could we just add this logic into the beginning of the original `case 0`?

Reply:
How's this? 

https://github.com/zcoop98/bingo64/blob/591ca626464eb5acd242187f83596567b302f090/src/game/behaviors/scuttlebug.inc.c#L36-L41

I still need a way to set the `BingoId` on first loop and never again, so I dropped the `break`.
This should mean that execution now goes straight into the original step 0, but still never hits the _new_ step 0 again.

Reply:
For reference, I got the idea from the [spindrift bingo code](https://github.com/matt-kempster/bingo64/blob/master/src/game/behaviors/spindrift.inc.c#L24-L29), which uses the same mechanism:

```c
switch (o->oAction) {
    case 0:
        o->oBingoId = get_unique_id(BINGO_UPDATE_KILLED_SPINDRIFT, o->oPosX, o->oPosY, o->oPosZ);
        o->oAction = 1;
        break;
    case 1:
```

Reply:
Ah wow you are TOTALLY right!

Yes, I very much approve of the `// fallthrough` comment, and am sad that I broke my own recommendation about the Spindrifts! Definitely think I should go back and fix that.